Sources/Fuzzilli/Environment/JavaScriptEnvironment.swift

```diff
@@ -835,6 +837,10 @@ public extension ObjectGroup {
             "flatMap"        : [.function(), .opt(.anything)] => .jsArray,
             "toString"       : [] => .jsString,
             "toLocaleString" : [.opt(.string), .opt(.object())] => .jsString,
+            "toReversed"     : [] => .jsArray,
+            "toSorted"       : [.opt(.function())] => .jsArray,
+            "toSpliced"      : [.integer, .opt(.integer), .anything...] => .jsArray,
+            "with"           : [.anything, .anything] => .jsArray,
```

Comment:
I think the first parameter should also be a `.integer` here since it should be an index IIUC

Reply:
Oh right, I think it should be `.integer`. Sorry I missed that earlier. I also missed that TypedArrays have a `with` method too, so I added that method's type information in the latest change